Dockerfile

```diff
@@ -11,9 +11,8 @@ SHELL	["/bin/bash", "-o", "pipefail", "-c"]
 
 # Install packages
 ARG	DEBIAN_FRONTEND=noninteractive
-RUN	apt-get update \
-&&	apt-get -y upgrade \
-&&	apt-get -y --no-install-recommends install $PACKAGES \
+RUN	apt-get -yfm --update --autoremove --purge full-upgrade \
+&&	apt-get -yfm --update --autoremove --purge --no-install-recommends install $PACKAGES \
```

Comment:
This does not work. There is no `--update` option for apt-get. At least in the current Debian 12 release.